util/wren_to_c_string.py

```diff
@@ -22,6 +22,7 @@
 def wren_to_c_string(input_path, wren_source_lines, module):
   wren_source = ""
   for line in wren_source_lines:
+    line = line.replace("\\", "\\\\")
```

Comment:
This should likely be it's own PR if Ruby has't fixed it already... I believe she mentioned it in another issue.  I've seen this also.

Reply:
I've used `\"` in `wren_core.wren` and which makes the  `wren_core.wren.inc` generation faill because `\"` become `\\"` and the c string terminates with the quote.

Reply:
_Yes, I'm familiar with the problem._  But this feature may or may not be merged - but that fix should be merged no matter what because it's actually a bug fix.

Reply:
For that line only yes.